src/Organisms/EnhancedList/EditColumns/EditColumns.jsx

```diff
@@ -34,24 +34,26 @@ const EditColumns = ({
   const disabledColumns = columns.filter(({ show }) => !show)
 
   return <Container>
-    <Title as="h2" fontSize="l" color="primary">
+    <Title>
       <ColumnsIcon size={24} />
-      <Trans transKey="global.editColumns.title" />
+      <Typo as="h2" fontSize="title" fontFamily="title" color="primary">
+        <Trans transKey="global.editColumns.title" />
+      </Typo>
     </Title>
-    <Typo>{descriptionText}</Typo>
+    <Typo as="div">{descriptionText}</Typo>
     {enabledColumns.length > 0 &&
       <ColumnGroup
         label={<Trans transKey="global.editColumns.enabledColumns"/>}
         columns={enabledColumns}
-        onChange={onChange}
+        onChange={name => onChange(name, false)}
         checked={true}
       />
     }
     {disabledColumns.length > 0 &&
       <ColumnGroup
-        label={<Trans transKey="global.editColumns.enabledColumns"/>}
+        label={<Trans transKey="global.editColumns.disabledColumns"/>}
         columns={disabledColumns}
-        onChange={onChange}
+        onChange={name => onChange(name, true)}
```

Comment:
So it's a toggle, no need to pass the value, it's just the opposite of whatever it was.

Reply:
harder to handle when we use the component